crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -118,17 +113,16 @@ Using the library
 
 Providing platform specific calloc/free
 ---------------------------------------
-This library facilitates the same type of memory management as regular Mbed TLS deliverables.
-This includes using internal calls to calloc/free from the library code when memory is needed.
+Just like Mbed TLS, this library calls calloc/free for memory management.
```

Comment:
- Is 'calloc/free' one function, or 2 separate functions? If they are 2 separate functions, then it should be rewritten to, "Just like Mbed TLS, this library calls the calloc or free for memory management."
- And if they are functions, then they need to be formatted differently (I believe like so, e.g., if they are C functions, then use `:c:func:´callout´`  `:c:func:´free´`).